bentoml/yatai/client/bento_repository_api.py

```diff
@@ -258,6 +285,7 @@ def upload_from_dir(self, saved_bento_path, labels=None):
     def _update_bento_upload_progress(
         self, bento_service_metadata, status=UploadStatus.DONE, percentage=None
     ):
+        # TODO separate update upload progress and bento metadata.
```

Comment:
I think it is ok leaving them in the same gRPC endpoint, we can also use it for adding and removing labels going forward

Reply:
remove the TODO?